lib/main_common.pm

```diff
@@ -457,6 +458,7 @@ sub load_zdup_tests {
     # Restrict version switch to sle until opensuse adopts it
     loadtest "migration/version_switch_upgrade_target" if is_sle and get_var("UPGRADE_TARGET_VERSION");
     loadtest 'boot/boot_to_desktop';
+    loadtest 'console/check_upgraded_service';
```

Comment:
I think this is pretty early. I see that could fail potentially for many different reasons which we try to fix with modules like "fixup", "consoletest_setup", etc. Could we move to let's say just before consoletest_finish?

Reply:
Feel free to do so or to open a ticket.
Getting in the main.pm this deep would take way too long for me at this point.